### DISCUSSION TITLE
I have a request to make to our instructor.

### DISCUSSION THREAD

**Discussion by anishkrjain** [akash-coded/spring-framework#54] [Q&A]
Hello Sir, 
I am facing a problem during the sessions . As we are not very fluent in writing codes so, it is becoming very challenging to understand the code on screen because of constant scrolling up and down of the screen. It will be easy for us to understand if the screen contains the whole code at a time and much scrolling is not happening. I am sharing a sample screenshot of clean and beginner friendly code. So, if more of us are facing the issue then you can look into this problem.

![display](https://user-images.githubusercontent.com/71398009/165468634-3fe9eea7-5dfa-4e06-89d3-606bd4deffd6.JPG)

**Comment by Anilkumar2306** [comment]
I am also facing the same problem

**Reply by akash-coded** [reply]
@anishkrjain @Anilkumar2306 Noted. ✅ Thank you for pointing this out. Will take measures to resolve this.

**Reply by anishkrjain** [reply]
Thank you Sir. Also, 2 or more codes can be put side by side if there is anything to compare between them. The output window and the project window may be minimized for sometime . Sir, I know it may be a problem for the instructor to cope up with beginners but it is a request. 

![display1](https://user-images.githubusercontent.com/71398009/165473713-61126154-1637-42fc-9645-faabf67b381a.JPG)

**Reply by akash-coded** [reply]
@anishkrjain Thank you, Anish. This will help everyone in the sessions I believe.
